pkg/model/components/context.go

```diff
@@ -120,13 +129,15 @@ func IsBaseURL(kubernetesVersion string) bool {
 
 // Image returns the docker image name for the specified component
 func Image(component string, clusterSpec *kops.ClusterSpec) (string, error) {
+
 	if component == "kube-dns" {
 		// TODO: Once we are shipping different versions, start to use them
-		return "gcr.io/google_containers/kubedns-amd64:1.3", nil
+		return GetGoogleImageRepositoryContainer(clusterSpec, "/kubedns-amd64:1.3"), nil
```

Comment:
Function could be named `GetDockerImage` ... I don't know if the fact that we expect it to be on gcr.io needs to be in the function name?